### PR TITLE
Fix formatting of `#each key=` special value list

### PR DESCRIPTION
**Pull request by mattmcmanus**
Noticed this was off when trying to make sense of the new #each abilities.

### CODE REVIEW

File: source/templates/displaying-a-list-of-items.md [hunk 1]
```diff
@@ -44,10 +44,11 @@ and related [Ember Mutable Array methods](http://emberjs.com/api/classes/Ember.M
...
   
   There are a few special values for `key`:
-    * `@index` - The index of the item in the array.
-    * `@item` - The item in the array itself.  This can only be used for arrays of strings
+  
+  * `@index` - The index of the item in the array.
+  * `@item` - The item in the array itself.  This can only be used for arrays of strings
       or numbers.
-    * `@guid` - Generate a unique identifier for each object (uses `Ember.guidFor`).
+  * `@guid` - Generate a unique identifier for each object (uses `Ember.guidFor`).
```
Comment (rwjblue) [inline source/templates/displaying-a-list-of-items.md]: This option is no longer present in 2.0.0-beta's.

File: source/templates/displaying-a-list-of-items.md [hunk 2]
```diff
@@ -44,10 +44,11 @@ and related [Ember Mutable Array methods](http://emberjs.com/api/classes/Ember.M
   property. Make sure the value you pass to `key` is unique!
   
   There are a few special values for `key`:
-    * `@index` - The index of the item in the array.
-    * `@item` - The item in the array itself.  This can only be used for arrays of strings
+  
+  * `@index` - The index of the item in the array.
+  * `@item` - The item in the array itself.  This can only be used for arrays of strings
```
Comment (rwjblue) [inline source/templates/displaying-a-list-of-items.md]: This option is no longer present in 2.0.0-beta's.

**Comment by rwjblue** [comment]
Can you remove `@guid` and `@item`?  Basically, the only special value that works in beta is `@index`, but it is generally unusual in 1.13.6+ to need to specify any key value (an `{{each` without key should "just do the right thing" now).

**Comment by mattmcmanus** [comment]
Oh right, I just noticed the different branches as well. Didn't realize master was for 2.0. I'll update this PR to fix the 1.13 docs and then one for master. 

Is `#each` with index (`{{#each people as |people index|}}`) still in 2.0?

**Comment by mattmcmanus** [comment]
Rereading the section, why not just remove that whole paragraph in favor of explaining the with index format introduced in `1.11`?

**Comment by locks** [comment]
@mattmcmanus yes, I believe index as a block param will still be available.
As for `key=@index` and the index block param, they are different things I believe. The first tells Ember what property to use to trigger state changes, the other is merely the index of the element in the array.

**Comment by michaelrkn** [comment]
@mattmcmanus We are cutting a release today. Will you have time to get to this in the next few hours? If not, no worries - I'll get this finished in a bit myself.

**Comment by mattmcmanus** [comment]
Hey @michaelrkn, I'll take a stab right now.

**Comment by mattmcmanus** [comment]
Alrighty! If you give it a once over for spelling/grammer/correctness. I think it's an improvement.

**Comment by rwjblue** [comment]
Thank you, that definitely reads much better to me. Can you squash commits?

**Comment by mattmcmanus** [comment]
Squashed.

**Comment by locks** [comment]
:boom: :hammer:

**Pull request merged**